compiler/statements.go

```diff
@@ -22,6 +22,27 @@ func (fc *funcContext) translateStmtList(stmts []ast.Stmt) {
 }
 
 func (fc *funcContext) translateStmt(stmt ast.Stmt, label *types.Label) {
+	defer func() {
+		err := recover()
+		if err == nil {
+			return
+		}
+		if _, yes := bailingOut(err); yes {
```

Comment:
Consider also using `ok` instead of `yes` here, that's what most Go code does, making it more readable.

Reply:
I am aware of this convention, but was reluctant to follow it because in this particular case the meaning of "ok" is unclear: is it ok when we are panicing or when we aren't? "Yes" answers the question more directly, so I settled on it.